docs(legal): adds legal section (#1742)
caos/zitadel: docs/docs/legal/support-services.md

---
title: Support Services
custom_edit_url: null
--- 

## Introduction

This annex of the [Framework Agreement](terms-of-service) describes the support services offered by us for our Services (ZITADEL Cloud).

> Support Services for products and services provided by CAOS is offered to customers according to the terms and conditions outlined in this document. The customer may purchase support services from CAOS AG directly.

Reviewer: ```suggestion
Support Services for products and services provided by CAOS is offered to customers according to the terms and conditions outlined in this document. The customer may purchase support services from CAOS Ltd. directly.
```

## Support Services

> **Business hours** means 08:00-17:00 Monday - Friday Switzerland time (or as per agreement with the customer). All times exclude public holidays in Switzerland / Kanton St. Gallen.

Reviewer: ```suggestion
**Business hours** means 08:00-17:00 Monday - Friday Switzerland time (or as per agreement with the customer). All times exclude public holidays in Switzerland / Canton St. Gallen.
```

eMail Support | n/a | yes | yes | yes
Chat Support | n/a | no | yes | yes
Phone Support | n/a | no | yes | yes
Technical Account Manager | n/a | n/a | n/a | n/a

### SLO - Initial response time
CAOS service level objective (SLO) for Support Services  is defined in terms of initial response time to a support request, as outlined in the table below per plan. CAOS will use reasonable efforts to resolve support requests, but does not guarantee a work-around, resolution or resolution time.

Subscription Plans | FREE | OUTPOST | STARBASE | FORTRESS
--- | --- | --- | --- | ---
Severity 1| not applicable | not applicable | 4 business hours | 1 business hours
Severity 2| not applicable | not applicable | 12 business hours | 2 business hours
Severity 3| not applicable | not applicable | 24 business hours | 12 business hours
Severity 4| not applicable | not applicable | 48 business hours | 24 business hours

### Communication
- Support is available in Swiss-German, German, and English
- Default contact: Whenever customers require support, Customers should consult the documentation of ZITADEL or post a question to our community.

> - When Customer is eligible for support services through a Subscription Plan, Customer may contact CAOS support via

Reviewer: ```suggestion
- When Customer is eligible for support services through a Subscription Plan, Customer may contact CAOS support via the following channels
```

--- | ---
eMail Support | support@zitadel.ch
Chat Support | Private chat channel between CAOS and Customer that is opened when Subscription becomes active
Phone Support | +41 43 215 27 34

- ZITADEL Cloud system status, incidents and maintenance windows will be communicated via https://status.zitadel.ch 
- Questions regarding pricing, billing, and invoicing of our services should be addressed to billing@caos.ch. 
- Security related questions and incidents can also be directly addressed to security@caos.ch.

### Technical account manager

CAOS will enhance its support offering by providing eligible clients with a Technical Account Manager (TAM), who will perform the following tasks for up to the specified amount of time per week during the term of service: 
- Provide support and advice regarding best practices on platform, product and configuration covered by the applicable Support Services;
- Participate in review calls every other week at mutually agreed times addressing customer’s operational issues.

*We currently offer TAM services only in the Enterprise plans. If you require consulting for your projects, please request a quote via hi@zitadel.ch*

> ## Support Features

Reviewer: ```suggestion
## Support
```

### Support request
CAOS agrees to handle support incidents in the following scenarios: 
1. ZITADEL Cloud software or configuration as provided by CAOS contains errors or critical security-related issues
2. ZITADEL Cloud requires upgrades or changes through the customer
3. ZITADEL Cloud have incorrect or missing documentation

> and support features include

Reviewer: ```suggestion
Support features include:
```

- Answer questions regarding usage of specific features or configurations of ZITADEL Cloud 
- Provide high-level suggestions regarding appropriate usage, features or configurations of ZITADEL Cloud functionality and configuration
- Assist in troubleshooting of issues to isolate potential root cause
- Document and advise alternative solutions for reported defects

> Excluded are broader consulting & customer-specific engineering requests regarding use of ZITADEL Cloud. Moreover first level support requests by Customer’s end users shall be handled by Customer directly.

Reviewer: ```suggestion
Excluded are broader consulting & customer-specific engineering requests regarding use of ZITADEL Cloud. Moreover first level support requests by Customer’s end users must be handled by the Customer directly.
```

### Support service process

The customer may submit support requests (“ticket”) through any means of eligible communication channels, consisting of
- Single discrete problem, issue, or request
- Initial severity level and impact statement for assessment
- Description of the issue and if possible a description of the observed and expected behavior,  steps to reproduce the issue, evidence that issue is not caused by connectivity / compute, relevant anonymized log-files etc.
- All information requested by CAOS as we resolve the ticket (e.g. system logs)

> CAOS  will review the case information and determine the severity level (see below), working with the customer to assess the urgency of the request and use reasonable efforts to respond to support requests within the initial response time 

Reviewer: ```suggestion
CAOS will review the case information and determine the severity level (see below), working with the customer to assess the urgency of the request and use reasonable efforts to respond to support requests within the initial response time 
```

CAOS will use reasonable efforts to resolve support request as defined below, but does not guarantee a workaround, resolution or resolution time

Severity Level | Description 
--- | --- 
**Severity 1**<br/> Critical / Service down| <p>Widespread failure or complete unavailability of CAOS Core Services. </p><p> CAOS will use continuous effort to provide a workaround or permanent solution. When Core Services are available, the severity will be lowered to the new appropriate level.</p>

> **Severity 2**<br/> Core functionality unavailable or severely degraded| <p>WCore Services of CAOS software continue to operate in severely restricted fashion, yet long-term productivity may be impacted.</p><p> When Core Services are no longer severely degraded (eg, through a viable workaround or release), the severity level will be lowered to Severity 3.</p>

Reviewer: ```suggestion
**Severity 2**<br/> Core functionality unavailable or severely degraded| <p>Core Services of CAOS software continue to operate in severely restricted fashion, yet long-term productivity may be impacted.</p><p> When Core Services are no longer severely degraded (eg, through a viable workaround or release), the severity level will be lowered to Severity 3.</p>
```

**Severity 3**<br/>Standard support request| <p>Partial and non-critical loss of CAOS software functionality or major software defect, yet a workaround exists for viable long-term operation.</p><p>CAOS will continue to work on developing permanent resolution.</p>
**Severity 4**<br/>Non-urgent request| <p>Defined as follows: <ul><li>Request for information or general query</li><li>Feature request</li><li>Performance issues and little to none functional impact</li><li>Defects with workarounds and little to low functional impact</li></ul></p>
<p>CAOS will continue to work on developing permanent resolution and response to general requests. CAOS does not provide a timeline or guarantee to include any feature requests.</p>

### Escalation

The customer may escalate support requests following the escalation process: 
1. For non-urgent needs, the client may request management escalation within the ticket. A manager will review the request and provide a response within one business day.

> 2. For urgent needs, the client may escalate directly by calling +41 43 456 84 69 and emailing to hi@caos.ch

Reviewer: were does this phone point to?

Author: This goes to my number. I think this is ok for now.

Reviewer: ok
